src/main/java/org/broadinstitute/hellbender/engine/ReferenceDataSource.java

```diff
@@ -64,6 +65,10 @@ default public ReferenceSequence queryAndPrefetch( final SimpleInterval interval
         return queryAndPrefetch(interval.getContig(), interval.getStart(), interval.getEnd());
     }
 
+    default ReferenceSequence queryAndPrefetch(final Interval interval) {
```

Comment:
Why not an overload that takes Locatable instead?

Reply:
Can do.

Reply:
will use queryAndFetch(String,int,int) instead.